test/views/vatDetails/VatDetailsViewSpec.scala

```diff
@@ -372,4 +404,26 @@ class VatDetailsViewSpec extends ViewBaseSpec {
       element(Selectors.header).attr("data-metrics") shouldBe "error:hidden-text:vat-overview-entity-name"
     }
   }
+
+  "Rendering the error message with error in Customer information API" should {
+
+    lazy val view = views.html.vatDetails.details(bothErrorDetailsModel)
+    lazy implicit val document: Document = Jsoup.parse(view.body)
+
+    "render Api graceful error message in Opt out section" in {
```

Comment:
Could you move this test to the clause above it which is regarding errors in all APIs. It constructs the view with the same view model you are using here so it should work the same.